test/ably/restchannelpublish_test.py

```diff
@@ -425,3 +432,19 @@ def test_idempotent_rest_publishing(self):
 
         ably = RestSetup.get_ably_rest(idempotent_rest_publishing=False)
         assert ably.options.idempotent_rest_publishing is False
+
+    # RSL1j
+    @dont_vary_protocol
+    def test_message_serialization(self):
+        channel = self.get_channel()
+
+        data = {
+            'name': 'name',
```

Comment:
Given this change is about idempotent publishing, are not checking that `id` carries through as well?